definitions/modules/autopilot.yaml

```diff
@@ -1,134 +1,275 @@
 shared:
-  -
+  - # ASOBO_AUTOPILOT_Push_Autopilot_Template
     type: ToggleSwitch
     var_name: A:AUTOPILOT MASTER
     var_units: Bool
-    var_type: bool
     event_name: AP_MASTER
+
+  - # ASOBO_AUTOPILOT_Push_Disconnect_Template
+    type: NumSet
+    var_name: A:AUTOPILOT DISENGAGED
+    var_units: Bool
+    var_type: i32
+    event_name: AUTOPILOT_DISENGAGE_SET
   -
+    type: ToggleSwitch
+    var_name: A:AUTOPILOT DISENGAGED
+    var_units: Bool
+    event_name: AUTOPILOT_DISENGAGE_TOGGLE
+
+  - # ASOBO_AUTOPILOT_Push_Heading_Template
     type: ToggleSwitch
     var_name: A:AUTOPILOT HEADING LOCK
     var_units: Bool
-    var_type: bool
     event_name: AP_PANEL_HEADING_HOLD
+  -
+    type: ToggleSwitch
+    var_name: A:AUTOPILOT HEADING LOCK
+    var_units: Bool
+    event_name: AP_HDG_HOLD
+
+  - # ASOBO_AUTOPILOT_Knob_Heading_Template
+    type: NumSet
+    var_name: A:HEADING INDICATOR
+    var_units: Degrees
+    var_type: f64
+    event_name: HEADING_BUG_SET
   -
     type: NumSet
     var_name: A:AUTOPILOT HEADING LOCK DIR
     var_units: Degrees
     var_type: f64
     event_name: HEADING_BUG_SET
+  -
+    type: event
+    event_name: HEADING_BUG_INC
+  -
+    type: event
+    event_name: HEADING_BUG_DEC
+
+  - # ASOBO_AUTOPILOT_Knob_Course_Template
+    type: NumSet
+    var_name: A:HEADING INDICATOR
+    var_units: Degrees
+    var_type: f64
+    event_name: VOR1_SET
+  -
+    type: NumSet
+    var_name: A:HEADING INDICATOR
+    var_units: Degrees
+    var_type: f64
+    event_name: VOR2_SET
+
+  - # ASOBO_AUTOPILOT_Push_Altitude_Template
+    type: ToggleSwitch
+    var_name: A:AUTOPILOT ALTITUDE LOCK
+    var_units: Bool
+    event_name: AP_PANEL_ALTITUDE_HOLD
   -
     type: ToggleSwitch
     var_name: A:AUTOPILOT ALTITUDE LOCK
     var_units: Bool
-    var_type: bool
     event_name: AP_ALT_HOLD
+
+  - # ASOBO_AUTOPILOT_Knob_Altitude_Template
+    type: NumSet
+    var_name: A:INDICATED ALTITUDE
+    var_units: Feet
+    var_type: i32
+    event_name: AP_ALT_VAR_SET_ENGLISH
   -
     type: NumSet
     var_name: A:AUTOPILOT ALTITUDE LOCK VAR
     var_units: Feet
     var_type: i32
     event_name: AP_ALT_VAR_SET_ENGLISH
+  -
+    type: event
+    event_name: AP_ALT_VAR_INC
+  -
+    type: event
+    event_name: AP_ALT_VAR_DEC
   -
     type: ToggleSwitch
     var_name: A:AUTOPILOT AIRSPEED HOLD
     var_units: Bool
     event_name: AP_PANEL_SPEED_HOLD
-  -
+
+  - # ASOBO_AUTOPILOT_Knob_Speed_Template
     type: NumSet
-    var_name: A:AUTOPILOT AIRSPEED HOLD VAR
+    var_name: A:AIRSPEED INDICATED
```

Comment:
again this doesn't seem right at all... why is the autopilot getting set from the airspeed indicated? it was fine as it was before.

Reply:
This didn't change, it just got moved, notice line number.

Reply:
regardless, this definition doesn't make sense. everytime the aircraft changes in speed the AP would be set to that speed.